framework-plugins/lisk-framework-dashboard-plugin/src/ui/utils/index.ts

```diff
@@ -20,7 +20,7 @@ interface Config {
 }
 
 const configDevEnvValues: Config = {
-	applicationUrl: 'ws://localhost:5000/ws',
+	applicationUrl: 'ws://localhost:8080/ws',
```

Comment:
shouldn't the port be read from config? if user change the default port and starts with different then faucet wont work?

Reply:
this is just default in the frontend, it will call API and overwrite it